app/src/main/kotlin/com/saltedge/authenticator/features/security/LockableActivity.kt

```diff
@@ -48,6 +48,7 @@ import com.saltedge.authenticator.widget.biometric.BiometricPromptAbs
 import com.saltedge.authenticator.widget.biometric.BiometricPromptCallback
 import com.saltedge.authenticator.widget.passcode.PasscodeInputView
 import com.saltedge.authenticator.widget.passcode.PasscodeInputViewListener
+import kotlinx.android.synthetic.main.view_passcode_input.*
```

Comment:
LockableActivity is abstract activity and does not have own layout.

Reply:
need to discuss